crew

```diff
@@ -927,6 +932,17 @@ def list_command (args)
   end
 end
 
+def postinstall_command (args)
+  args["<name>"].each do |name|
+    set_package name, true
+    if @device[:installed_packages].any? do |elem| elem[:name] == name end
+      @pkg.postinstall
+    else
+      puts "Package #{name} is not installed. :(".lightred
```

Comment:
I don't think it should require the package to be installed.

Reply:
Actually, it does in several instances where files are expected to exist.  When you think about it, it does make sense.

Reply:
Yes, it could also overwrite user configuration. Just output a warning if the package isn't installed.

Reply:
That's what I'm doing.

Reply:
Yes, but you should still go through with the command. You know, there's a difference between a warning and an error.

Reply:
No, I shouldn't.  See above.

Reply:
All right. Other than that, this is fine.

Reply:
Is there any way you could add an option to force it? I've been looking forward to this for a long time.

Reply:
Why would you need to run postinstall on an uninstalled package?

Reply:
Testing a new package and making sure you haven't forgotten `.lightblue` without installing it? Making sure there aren't any typos?

Reply:
You wouldn't see that at the end anyway?

Reply:
I don't want to go through the whole build process again just to see the postinstall when I've changed it.

Reply:
Wait, testing a package includes installing it which would also output the text at the end.  You really should be testing it with the install anyway.

Reply:
I don't want to build a package again when all I've changed is the postinstall.